### ISSUE TITLE
Think about abbeel

### ISSUE THREAD

**Issue by RaulFdzbis** [roboticslab-uc3m/xgnitive#15]
\juansays{Think about abbeel}

\davidsays{![Abbeel](http://vcresearch.berkeley.edu/sites/default/files/abbeel.jpg)}
- [x] 1/3
- [ ] 2/3
- [ ] 3/3

**Comment by jgvictores** [comment]
I've given sparse insight on this in #18, and added some checkboxes to the original issue to see some progress.

 @David-Estevez Do you think this deserves a milestone?

**Comment by David-Estevez** [comment]
Only if you have clear in your mind what 1/3 2/3 and 3/3 mean.

**Comment by jgvictores** [comment]
I believe they are rational numbers. Perhaps we could address the non-linearities of the thought process through a taylor series expansion or an unscented transform?

**Comment by David-Estevez** [comment]
Due to the non-linear nature of the process an stochastic optimization of the quadrature function with an heuristic approximation by a divide-and-conquer approach will suffice.

**Comment by jgvictores** [comment]
This is good, because another one bites the dust. However, we should also be concerned about security issues. I'd look into creating a Merkle tree to prevent tampering of the data.
cc: @smorante Could you provide any insight or ideas?

**Comment by RaulFdzbis** [comment]
I didnt understood a word of what you have said @David-Estevez , so i suppose you are right. You have my support in this.

**Comment by David-Estevez** [comment]
Yeah, don't stop me now, cause I'm having a good time ( having a good time).

**Comment by jgvictores** [comment]
After a night at the opera, I've ticked 1/3 thanks to the DRL course which really made me work on this issue.
